### ISSUE TITLE
List operations: handle or prevent operations with multiple lists

### ISSUE THREAD

**Issue by yochannah** [intermine/bluegenes#20]
Right now (I believe) the combine, difference, and intersect buttons expect exactly two lists and will cause odd behaviour if there are more than two selected. We need to either:

a) prevent operations on more than two lists   
b) make it so that these operations work as expected. 

I suspect a) will be much easier and is probably the saner short term solution.

**Comment by yochannah** [comment]
I may have been lying about this; it might actually already work. need to test.

**Comment by joshkh** [comment]
I do suspect it's working. The function signatures for `im-list-union` `im-list-intersect` `im-list-difference` expect `lists` to be a collection:

```clojure
(defn im-list-union
  [service name lists & [options]]...
```
https://github.com/intermine/imcljs/blob/master/src/imcljs/save.cljs#L22

When I tested the functions I created a list of 10 genes. Then created a second list from the first six, and a third list from the last six.

```
L1) A B C D E F G H I J
L2) A B C D E F
L3)         E F G H I J
```

Union L2 and L3 = 10 (L1)
Intersection L2 and L3 = 2 (E F)
Difference L2 L3 = 8 (L1 - E F)

Asymmetric difference only supports two lists by the web service, and copying / deleting supports just one. For copying and deleting multiple lists in bluegenes I just fire the requests in a loop!
